types/nationalAvalancheCenter/schemas.ts

```diff
@@ -31,21 +31,21 @@ export const AvalancheCenterWebsites: Record<AvalancheCenterID, string> = {
   ['NWAC']: 'https://nwac.us/',
   ['BAC']: 'https://bridgeportavalanchecenter.org/',
   ['BTAC']: 'https://bridgertetonavalanchecenter.org/',
-  ['CBAC']: 'https://cbavalanchecenter.org/',
-  ['CNFAIC']: 'https://www.cnfaic.org/',
+  ['CBAC']: 'https://cbavalanchecenter.org/view-', // special link for obs sharing https://cbavalanchecenter.org/view-observations/#/view/observations/ddc9783a-a321-45cd-b656-88b066e5868d
```

Comment:
still need your thoughts on what the best way to do this @stevekuznetsov  -- right now I am learning towards just making a file where I hardcode how to construct these to display the right obs link on the Observation page and just leave the links in here as the hostname. Figured that may just be easiest.

Reply:
React Navigation fundamentally does not allow for more than one route to be mapped to a specific screen, and that is exactly what we need to do. I unfortunately think that either:

- if `getStateFromPath()` gives you the entire path with the host name, we can write a function to transform any given URL for any center into the correct React Navigation screen state
- if it does not, we will need to drop down into the bare-bones listener approach.

Reply:
ah good point, I was only really thinking of this in the context of building out the link - not the prefix part....let me think about that part a bit more

Reply:
decided that for now we will ignore prefixes and how we build them since none of the other centers are ready to have their obs open in the app anyways.